tests/cnf/core/network/sriov/tests/externalllymanaged.go

```diff
@@ -271,7 +275,9 @@ var _ = Describe("ExternallyManaged", Ordered, Label(tsparams.LabelExternallyMan
 				Expect(err).ToNot(HaveOccurred(), "VFs were removed after SR-IOV operator reinstallation")
 
 				By("Configure SR-IOV with flag ExternallyManage true")
-				createSriovConfiguration(sriovAndResourceNameExManagedTrue, sriovInterfacesUnderTest[0], true)
+				err = sriovenv.CreateSriovNmStateConfiguration(sriovAndResourceNameExManagedTrue, sriovInterfacesUnderTest[0], true)
+				Expect(err).ToNot(HaveOccurred(),
+					fmt.Sprintf("Failed to create sriov configuration via nmstate %s", err))
```

Comment:
wrong discription and remove err:
Expect(err).ToNot(HaveOccurred(),"Failed to create sriov configuration with flag ExternallyManaged true")

Reply:
changed